x/stakeibc/ibc_middleware.go

```diff
@@ -90,52 +91,93 @@ func (im IBCMiddleware) OnChanOpenAck(
 	if err != nil {
 		return err
 	}
-	delegationAddress, err := icatypes.NewControllerPortID(types.FormatICAAccountOwner(hostChainId, types.ICAAccountType_DELEGATION))
+	delegationPortID, err := icatypes.NewControllerPortID(types.FormatICAAccountOwner(hostChainId, types.ICAAccountType_DELEGATION))
 	if err != nil {
 		return err
 	}
-	redemptionAddress, err := icatypes.NewControllerPortID(types.FormatICAAccountOwner(hostChainId, types.ICAAccountType_REDEMPTION))
+	redemptionPortID, err := icatypes.NewControllerPortID(types.FormatICAAccountOwner(hostChainId, types.ICAAccountType_REDEMPTION))
 	if err != nil {
 		return err
 	}
 	communityPoolDepositOwner := types.FormatICAAccountOwner(hostChainId, types.ICAAccountType_COMMUNITY_POOL_DEPOSIT)
-	communityPoolDepositAddress, err := icatypes.NewControllerPortID(communityPoolDepositOwner)
+	communityPoolDepositPortID, err := icatypes.NewControllerPortID(communityPoolDepositOwner)
 	if err != nil {
 		return err
 	}
 	communityPoolReturnOwner := types.FormatICAAccountOwner(hostChainId, types.ICAAccountType_COMMUNITY_POOL_RETURN)
-	communityPoolReturnAddress, err := icatypes.NewControllerPortID(communityPoolReturnOwner)
+	communityPoolReturnPortID, err := icatypes.NewControllerPortID(communityPoolReturnOwner)
 	if err != nil {
 		return err
 	}
 
 	// Set ICA account addresses
 	switch {
-	case portID == withdrawalAddress:
+	case portID == withdrawalPortID:
 		zoneInfo.WithdrawalIcaAddress = address
-	case portID == feeAddress:
+	case portID == feePortID:
 		zoneInfo.FeeIcaAddress = address
-	case portID == delegationAddress:
+	case portID == delegationPortID:
 		zoneInfo.DelegationIcaAddress = address
-	case portID == redemptionAddress:
+	case portID == redemptionPortID:
 		zoneInfo.RedemptionIcaAddress = address
-	case portID == communityPoolDepositAddress:
+	case portID == communityPoolDepositPortID:
 		zoneInfo.CommunityPoolDepositIcaAddress = address
-	case portID == communityPoolReturnAddress:
+	case portID == communityPoolReturnPortID:
 		zoneInfo.CommunityPoolReturnIcaAddress = address
 	default:
 		ctx.Logger().Error(fmt.Sprintf("Missing portId: %s", portID))
 	}
 
+	// Once the delegation channel is registered, whitelist epochly transfers so they're not rate limited
+	// Epochly transfers go from the deposit address to the delegation address
+	if portID == delegationPortID {
+		im.keeper.RatelimitKeeper.SetWhitelistedAddressPair(ctx, ratelimittypes.WhitelistedAddressPair{
+			Sender:   zoneInfo.DepositAddress,
+			Receiver: zoneInfo.DelegationIcaAddress,
+		})
+	}
+
+	// Once the fee channel is registered, whitelist reward transfers so they're not rate limited
+	// Reward transfers go from the fee address to the reward collector
+	if portID == feePortID {
+		rewardCollectorAddress := im.keeper.AccountKeeper.GetModuleAccount(ctx, types.RewardCollectorName).GetAddress()
+		im.keeper.RatelimitKeeper.SetWhitelistedAddressPair(ctx, ratelimittypes.WhitelistedAddressPair{
```

Comment:
To confirm, setting whitelisted address pairs works both when the sender is on stride and the receiver is off stride or when the sender if off stride and the receiver is on stride?

Reply:
yup!